nodes/liveplot

```diff
@@ -0,0 +1,211 @@
+        self._super_packet = None
+        rospy.Subscriber("/flydra_mainbrain/super_packets",
+                         ros_flydra.msg.flydra_mainbrain_super_packet,
+                         self._on_target_move)
+
+    def _on_target_move(self, msg):
+        self._super_packet = msg.packets[0]
+
+    def get_super_packet(self):
+        return self._super_packet
+
+
+class DataHandler():
+    '''
+    we store xzy for every objid with this class to a circular buffer
+    new objIds are added, old ones that are not updated for a periode of time
+    are deleted
+    '''
+    def __init__(self, buffer_length=1000, dead_after_secs=30):
+        self.obj = {}  # memory
+        self.buffer_length = buffer_length
+        self.dead_after_secs = dead_after_secs
+        self._super_packet = None
+        rospy.Subscriber("/flydra_mainbrain/super_packets",
+                         ros_flydra.msg.flydra_mainbrain_super_packet,
+                         self._on_target_move)
+
+    def _on_target_move(self, msg):
+        self._super_packet = msg.packets[0]
+
+    def aquire(self):
+        '''
+        update data of obj
+        '''
+        new_dataset = self._super_packet
+        if new_dataset != None:  # is none if nothing comes from ros
+            teimestamp = new_dataset.acquire_stamp.secs
+            objects = new_dataset.objects
+            # add data from objects
+            for obj in objects:
+                # if new obj
+                if obj.obj_id not in self.obj:  # add ob to dict if not existing
+                    self.obj[obj.obj_id] = {}
+                    self.obj[obj.obj_id]['x'] = collections.deque(
+                                        [obj.position.x] * self.buffer_length,
+                                        maxlen=self.buffer_length
+                                        )
+                    self.obj[obj.obj_id]['y'] = collections.deque(
+                                        [obj.position.y] * self.buffer_length,
+                                        maxlen=self.buffer_length
+                                        )
+                    self.obj[obj.obj_id]['z'] = collections.deque(
+                                        [obj.position.z] * self.buffer_length,
+                                        maxlen=self.buffer_length
+                                        )
+                self.obj[obj.obj_id]['lastSeen'] = teimestamp
+                self.obj[obj.obj_id]['x'].append(obj.position.x)
+                self.obj[obj.obj_id]['y'].append(obj.position.y)
+                self.obj[obj.obj_id]['z'].append(obj.position.z)
+            # cleanup old objids
+            for obj_id in self.obj.keys():
+                time_dif = teimestamp - self.obj[obj_id]['lastSeen']
+                if time_dif >= self.dead_after_secs:  # LAMEST FIX IT!
+                    del self.obj[obj_id]
+
+    def get_max_min(self):
+        all_x = list(itertools.chain(*[self.obj[i]['x'] for i in self.obj]))
```

Comment:
min_max = {}
    for a in 'xyz':
        v = list(itertools.chain(*[self.obj[i][a] for i in self.obj]))
        min_max['min_%s' % a] = min(v)
        min_max['max_%s' % a] = max(v)